src/index.js

```diff
@@ -6,5 +6,10 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
-// The primary React component to use.
-module.exports = require('./components/GraphiQL').GraphiQL;
+import { GraphiQL } from './components/GraphiQL';
+import { QueryEditor } from './components/QueryEditor';
+import { VariableEditor } from './components/VariableEditor';
+import { ResultViewer } from './components/ResultViewer';
+
+export default GraphiQL;
+export { QueryEditor, VariableEditor, ResultViewer };
```

Comment:
Why re-export all of this stuff which is already on the default `GraphiQL` export anyway? We haven't done that with the other component properties, so this is inconsistent.